mathesar_ui/src/systems/table-view/table-inspector/record/RowActions.svelte

```diff
@@ -1,15 +1,25 @@
 <script lang="ts">
-  import { Button, Icon, iconLoading } from '@mathesar/component-library';
-  import { iconDeleteMajor } from '@mathesar/icons';
+  import {
+    AnchorButton,
+    Button,
+    Icon,
+    iconExternalLink,
+    iconLoading,
+  } from '@mathesar/component-library';
```

Comment:
@rajatvijay Please ensure that all imports of the component library from mathesar code happens through `@mathesar-component-library` alias. Automatic imports would prefer directly accessing the directory directly and we do not have a way to force the alias as of the moment, expect manually.

Reply:
No change needed in this PR, we can correct the existing imports later. However, please ensure to follow this in upcoming PRs.